ui/desktop/src/main.ts

```diff
@@ -659,25 +659,42 @@ const createChat = async (
       .executeJavaScript(
         `
```

Comment:
A bit outside the scope of this PR, but why do we need to manually call `executeJavaScript` on this block?

Could we not move it to some renderer side ts?